StyleCop.Analyzers/StyleCop.Analyzers/Helpers/XmlCommentHelper.cs

```diff
@@ -18,8 +18,9 @@ internal static class XmlCommentHelper
         internal const string InheritdocXmlTag = "inheritdoc";
         internal const string ReturnsXmlTag = "returns";
         internal const string ValueXmlTag = "value";
-        internal const string SeeXmlTag = "see";
-        internal const string ParamTag = "param";
+        internal const string SeeTag = "see";
```

Comment:
:question: Why did you change `SeeXmlTag` &rarr; `SeeTag`, but `ParamTag` &rarr; `ParamXmlTag`?